src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -235,6 +241,25 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 	public override isDirty(): boolean { return this._text.isDirty(); }
 	public get resource(): URI { return this._text.resource; }
 
+	public async getServerMetadata(): Promise<string[]> {
+		const copilotExt = await this.extensionService.getExtension('github.copilot');
+
+		if (copilotExt && this.configurationService.getValue<IQueryEditorConfiguration>('queryEditor').githubCopilotContextualizationEnabled) {
+			if (!this._databaseServerContext) {
+				const result = await this.databaseServerContextualizationService.getDatabaseServerContextualization(this.uri);
```

Comment:
Yeah, in the future we will most like make adjustments, but for now it was suggested that we don't need the most up to date data, which why there's a 30 day period before scripts are generated again for a server.

Reply:
Have you considered at least adding a "Contextualization Refresh" command or something which will clear the cached items? That will probably be something we want eventually, and would at least give people an option (besides making a new query) to get new data. (this would be something for a follow up PR, just throwing out ideas)